Riot/Modules/Analytics/Analytics.swift

```diff
@@ -276,18 +278,27 @@ extension Analytics: MXAnalyticsDelegate {
         capture(event: event)
     }
     
-    func trackJoinedRoom(asDM isDM: Bool, memberCount: UInt) {
+    func trackJoinedRoom(asDM isDM: Bool, isSpace: Bool, memberCount: UInt) {
         guard let roomSize = AnalyticsEvent.JoinedRoom.RoomSize(memberCount: memberCount) else {
             MXLog.warning("[Analytics] Attempt to capture joined room with invalid member count: \(memberCount)")
             return
         }
         
-        let event = AnalyticsEvent.JoinedRoom(isDM: isDM, roomSize: roomSize)
+        let event = AnalyticsEvent.JoinedRoom(isDM: isDM, isSpace: isSpace, roomSize: roomSize, trigger: nil)
         capture(event: event)
     }
     
     /// **Note** This method isn't currently implemented.
     func trackContactsAccessGranted(_ granted: Bool) {
         // Do we still want to track this?
     }
+
+    func trackEventComposed(inThread: Bool, isEditing: Bool, isReply: Bool, startsThread: Bool) {
+        let event = AnalyticsEvent.Composer(inThread: inThread,
```

Comment:
Given the event name is `Composer` it might align a little more if this method was `trackComposerInteraction…`, but I'm splitting hairs here.

Reply:
Hmm this is actually not an interaction, in terms of our analytics. `trackComposer` seems more suitable to be consistent with other method names, but it doesn't make sense much :) How about `trackComposerEvent`?